app/src/main/java/com/amaze/filemanager/ui/dialogs/GeneralDialogCreation.java

```diff
@@ -162,15 +162,7 @@ public static MaterialDialog showNameDialog(
     WarnableTextInputLayout tilTextfield =
         dialogView.findViewById(R.id.singleedittext_warnabletextinputlayout);
 
-    tilTextfield.requestFocus();
-
-    textfield.postDelayed(
-        () -> {
-          InputMethodManager inputMethodManager =
-              (InputMethodManager) m.getSystemService(Context.INPUT_METHOD_SERVICE);
-          inputMethodManager.showSoftInput(textfield, InputMethodManager.SHOW_IMPLICIT);
-        },
-        100);
+    dialogView.post(() -> ExtensionsKt.openKeyboard(textfield, m));
```

Comment:
Had to do this, because it didn't occasionally work for me. Found [this](https://stackoverflow.com/a/36228770/9652621) that explains why it might behave so (the content view might not have been created when `requestFocus()` is being called). So, found calling from `rootView.post` would be the best bet.